daemon/daemon.go

```diff
@@ -42,17 +42,19 @@ const (
 // Daemon is the fully-functional state of a daemon (compare to
 // `NotReadyDaemon`).
 type Daemon struct {
-	V              string
-	Cluster        cluster.Cluster
-	Manifests      cluster.Manifests
-	Registry       registry.Registry
-	ImageRefresh   chan image.Name
-	Repo           *git.Repo
-	GitConfig      git.Config
-	Jobs           *job.Queue
-	JobStatusCache *job.StatusCache
-	EventWriter    event.EventWriter
-	Logger         log.Logger
+	V                        string
+	Cluster                  cluster.Cluster
+	Manifests                cluster.Manifests
+	Registry                 registry.Registry
+	ImageRefresh             chan image.Name
+	Repo                     *git.Repo
+	GitConfig                git.Config
+	lastKnownSyncTagRev      string
```

Comment:
Is there any reason to stick this in the struct, rather than it being local to the loop? (You'd have to pass it to doSync maybe)

Reply:
Fair point, no real reason, no. I will change it.

Reply:
Done, PTAL